testsuite/python/lb_interpolation.py

```diff
@@ -72,9 +72,21 @@ def test_interpolated_velocity(self):
         """
         self.set_boundaries([0.0, 0.0, V_BOUNDARY])
         self.system.integrator.run(250)
-        # Shear plane for boundary 1
-        # for pos in itertools.product((AGRID,), np.arange(0.5 * AGRID, BOX_L, AGRID), np.arange(0.5 * AGRID, BOX_L, AGRID)):
-        #     np.testing.assert_almost_equal(self.lbf.get_interpolated_velocity(pos)[2], 0.0)
+        # Check interpolated vel at upper boundary. The node position is at 
+        # box_l[0]-agrid/2.
+        np.testing.assert_allclose(self.lbf.get_interpolated_velocity(
```

Comment:
do we need to guard the result of `get_interpolated_velocity` with `np.copy()`?

Reply:
We let the CI decide.